components/Ui/UiCta.vue

```diff
@@ -221,9 +221,11 @@ function isIdAnchor(url: string) {
     &#{$arrow-right-path} {
       transform: translate(4px, 0);
     }
+
```

Comment:
is this change coming from a lint issue?

Reply:
yes, linter shows this solution